main.py

```diff
@@ -1,44 +1,39 @@
-import js
 import libcst as cst
 
 from functools import singledispatchmethod, singledispatch
 from contextlib import contextmanager
 
-#########
-#  Current State
-########
-
+from pyweb import pydom
+from pyodide.ffi.wrappers import add_event_listener
+from pyscript.js_modules.monaco import editor
+from pyscript import document
+from pyscript.ffi import create_proxy, to_js
+from js import localStorage, Array
+
+DEFAULT_CONTENT = """
+def foo[T](bar: T | None = None) -> T | None:
+    if bar is None:
+        return None
+    # increment
+    return bar + 1
+"""
+EDITOR_CONTENT_KEY = "libcst-me-editor-content"
```

Comment:
this used to be `editor-content`. Not sure if we care about preserving local storage